_Week8/Group3/results.do

```diff
@@ -0,0 +1,36 @@
+clear all
+
+* Using code I saw in Group 4's code to set the working directory!
+
+global username "/Users/anton/OneDrive/Documents/Georgetown/ECON/ECON490" // you have to change the line above
```

Comment:
```suggestion
global username "/Users/anton/OneDrive/Documents/Georgetown/ECON/ECON490" // you have to change the line above to the path preceding the repository
```

As a note, be careful storing Git repos in sync folders like OneDrive -- this may cause problems! I recommend in general to keep those separate (I know it feels weird since this is a class assignment repo but organizing your repos in their own location is a good habit to get into!)